src/ledger.ts

```diff
@@ -909,6 +913,26 @@ export class LedgerExportExtendedPublicKey extends LedgerExportHDNode {
   }
 }
 
+interface LedgerSignMultisigTransactionArguments {
+  network: BitcoinNetwork;
+
+  inputs: TxInput[];
+
+  outputs: object[];
```

Comment:
I think in typescript `object` is the type of an empty object (`{}`). As you can imagine, it's not a very useful type. Unless this interface will always receive outputs as `{}[]` (read as an array of empty objects), it should be `any[]`.

Leaving it as is will error when you pass it something like `[{ value: true }]` saying something like `Type object has no attribute "value"`

Reply:
I feel like I've seen this work as `object` but i could be wrong. I've updated to be `outputs: Record<string, unknown>[];` in any case until we can get better ts penetration.

Reply:
Seems to work, but doesn't hurt to change to a record in any case. 

![Screenshot 2023-02-21 at 6 02 27 PM](https://user-images.githubusercontent.com/4344978/220486345-019d198b-69df-44d9-9051-a7a4ec1ce803.png)

Reply:
Perhaps you are right. I'm looking at reference for it now. I could be confusing `object` with `Object`.